ArcGISRuntimeSDKQt_QMLSamples/Maps/ShowLocationHistory/ShowLocationHistory.qml

```diff
@@ -0,0 +1,115 @@
+// [WriteFile Name=ShowLocationHistory, Category=Maps]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+import Esri.ArcGISRuntime 100.8
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    readonly property int initialZoomScale: 10000
+    property bool trackingEnabled: false
+    property Point lastPosition: null
+
+    // add a mapView component
+    MapView {
+        anchors.fill: parent
+
+        Map {
+            // add the BasemapImageryWithLabelsVector basemap to the map
+            BasemapDarkGrayCanvasVector {}
+        }
+
+        Button {
+            id: button
+            text: trackingEnabled ? "Stop tracking" : "Start tracking"
+            anchors.horizontalCenter: parent.horizontalCenter
+            width: 200
+            onClicked: trackingEnabled = !trackingEnabled
+        }
+
+        locationDisplay {
+            dataSource: SimulatedLocationDataSource {
+                id: simulatedLocationDataSource
+                Component.onCompleted: {
+                    setLocationsWithPolyline(GeometryEngine.densify(polyline, 10));
```

Comment:
I'm not sure if you need to densify. i think the simulated location data source (or it's parameters) will interpolate new points depending on the velocity you set - i may be wrong and this is fine as you have it